pkg/cmd/api/api.go

```diff
@@ -175,6 +175,11 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 			opts.RequestPath = args[0]
 			opts.RequestMethodPassed = c.Flags().Changed("method")
 
+			if strings.HasPrefix(opts.RequestPath, "/") {
+				warningIcon := opts.IO.ColorScheme().WarningIcon()
+				fmt.Fprintf(opts.IO.ErrOut, "%s warning: '%s' An absolute window-styled path was passed which is likely to fail. Try dropping or escaping the leading slash in the endpoint if it fails. \n", warningIcon, opts.RequestPath)
```

Comment:
Hi, I think you might have misunderstood the solution I've proposed in the issue.

The problem isn't that some users pass in a path argument with a leading slash. That style of usage is perfectly valid and we shouldn't warn against it.

The problem is that the mingw environment converts those arguments with a leading slash into an absolute, Windows-style file path before invoking `gh`, which then breaks the `gh api` command. When that happens, we should error out explaining to the user what happened.

So, what we can do is:
1. Check whether the argument is a Windows-style absolute path. This can be done with `runtime.GOOS == "windows" && filepath.IsAbs(opts.RequestPath)`
2. Error out and explain to the user that they shouldn't use leading slash in their environment.

Think you could do it?